module/entities/helper.js

```diff
@@ -480,4 +480,49 @@ module.exports = class EntitiesHelper {
     })
   }
 
+   /**
+   * Entity details information.
+   * @method 
+   * @name details
+   * @param {String} entityId - _id of entity.
+   * @return {Object} - consists of entity details information. 
+   */
+
+  static details( entityId ) {
+    return new Promise(async (resolve, reject) => {
+        try {
+
+            let entityDocument = await this.entityDocuments(
```

Comment:
@aman-tunerlabs - Can we please put negative condition here, i.e. everything except groups please

Reply:
@aks30  i got your point and thought the same instead of projecting everything just project groups : 0 but i am referring to entityDocuments method where we are expecting an array and making something like {field : 1}

Reply:
@aman-tunerlabs - yes so modify that to take array of fields which if given will be skipped

Reply:
Ok